flask_monitoringdashboard/__init__.py

```diff
@@ -29,12 +29,13 @@ def loc():
 blueprint = Blueprint('dashboard', __name__, template_folder=loc() + 'templates')
 
 
-def bind(app, schedule=True):
+def bind(app, schedule=True, micro=False):
     """Binding the app to this object should happen before importing the routing-
     methods below. Thus, the importing statement is part of this function.
 
     :param app: the app for which the performance has to be tracked
     :param schedule: flag telling if the background scheduler should be started
+    :param micro: flag telling if the views should be added - default False - True if you want only the db stuff
```

Comment:
I think this docstring deserves an update:
- ` - default False - `: can be removed.
- `True if you only want the db stuff` doesn't sound really professional to me :)

Reply:
Fair. removed.